__tests__/chat.test.ts

```diff
@@ -202,7 +220,7 @@ describe('Chat Methods', () => {
         },
         {name: `${config.bots.alice1.username},${config.bots.bob1.username}`},
       ]
-      const okChecks = []
+      const okChecks: any[] = []
```

Comment:
can `okChecks` be typed as a boolean array?

Reply:
yeah I think so